### PR TITLE
Implemented `AstoAuthTokens.generate()`

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #57 
Implemented  and tested`AstoAuthTokens.generate()` to add new tokens.

### CODE REVIEW

File: src/main/java/com/artipie/conda/asto/AstoAuthTokens.java [hunk 1]
```diff
@@ -79,7 +98,35 @@ public AstoAuthTokens(final Storage asto, final String ttl) {
 
     @Override
     public CompletionStage<String> generate(final String name) {
-        return null;
```
Comment (genryxy) [inline src/main/java/com/artipie/conda/asto/AstoAuthTokens.java:82]: @olenagerasimova probably to pass `ttl` as a parameter to this method and change type from `String` to `Duration`

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, one suggestion

**Comment by olenagerasimova** [comment]
@genryxy thanks, corrected as you suggested

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks, looks good to me

**Pull request merged**